jaxrs-tck/pom.xml

```diff
@@ -18,26 +18,26 @@
 -->
 
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>${tck.artifactId}</artifactId>
+    <artifactId>jakarta-restful-ws-tck</artifactId>
+    <name>Jakarta RESTful WS TCK</name>
+    <description>Technology Compatibility Kit for Jakarta RESTful Web Services</description>
+    <url>https://github.com/eclipse-ee4j/jaxrs-api</url>
```

Comment:
The URL is now gone

Reply:
Technically it's not gone, but it redirects. Indeed better to put https://github.com/jakartaee/rest there.